### ISSUE TITLE
# Add Remedy Form Update

### ISSUE THREAD

**Issue by SylvieDoumet** [Rahaf91/capstone-natural-cures#26]
**Value proposition**

As a User I want to have a button to open the Add Remedy Form in a new window. In order to keep the landing page clean and not stuffed with too many different input fields and informations. 

**Acceptance criteria**

- The button is fixed within the header component
- Clicking the Add Remedy button opens a page showing the Submit form.
- Cancel button and Submit button close the Form. The user is redirected to the main page (RemediesList).

**Tasks**

- [ ] Create a feature branch `feature/Add-remedy-formupdate`
- [ ] Create AddRemedy_Button component
- [ ] Write a onClick-function to open the Add Remedy_Form component as page
- [ ] Add a Cancel and Submit button to close the  Add Remedy_Form  and a onClick-function to redirect the User 
       to RemediesList
- [ ] Re-use AddRemedy component

**Comment by Inelukas** [comment]
Overall, the user story looks fine!

I would rephrase the value preposition. The coaches often tell us that as a user you don't want a certain html element but an improved experience. One possibility would be as follows:

"As a user, I want to easily open the 'Add Remedy' form in a separate window, in order to keep the landing page uncluttered."

You could also specify if the button should be fixed or not. Currently, the header disappears when scrolling down so the button would disappear as well.
